steps/appointee/same-address/SameAddress.js

```diff
@@ -23,10 +24,12 @@ class SameAddress extends SaveToDraftStore {
   }
 
   answers() {
+    const content = require(`./content.${i18next.language}`);
+
     return answer(this, {
       question: this.content.cya.isAddressSameAsAppointee.question,
       section: sections.appellantDetails,
-      answer: titleise(this.fields.isAddressSameAsAppointee.value)
+      answer: titleise(content.cya.isAddressSameAsAppointee[this.fields.isAddressSameAsAppointee.value])
```

Comment:
What about content.en? it seems only done for cya

Reply:
Not sure what you mean

Reply:
Oh I think I understand what you're saying.
It's CYA (Check Your Appeal) - not CY (Welsh).
As you can see, the correct `content` (EN or CY) gets loaded dynamically on line 27.
Then in line 32 it takes the `.cya.isAddressSameAsAppoint` property from it and matches it to what the user answered (`yes` or `no`).

Reply:
ah ok, its confusing content.cya and content.cy, can we please not change to relevant